cmake/external/spdlog.cmake

```diff
@@ -1,20 +1,45 @@
 SET(spdlog_SOURCE_DIR ${THIRD_PARTY_DIR}/spdlog)
 SET(spdlog_INCLUDE_DIR ${THIRD_PARTY_INCLUDE_DIR})
+set(spdlog_PREFIX "${CMAKE_BINARY_DIR}/spdlog")
+set(spdlog_DIR "${spdlog_PREFIX}/src/SpdLog-build")
+IF(CMAKE_BUILD_TYPE MATCHES Debug)
+	set(spdlog_LIBRARY_NAME ${spdlog_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}spdlogd${CMAKE_STATIC_LIBRARY_SUFFIX})
+else()
+	set(spdlog_LIBRARY_NAME ${spdlog_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}spdlog${CMAKE_STATIC_LIBRARY_SUFFIX})
+endif()
+
 include(ExternalProject)
 ExternalProject_Add(
-	spdlog
-	PREFIX ${CMAKE_BINARY_DIR}/spdlog
+	SpdLog
+	PREFIX ${spdlog_PREFIX}
 	DOWNLOAD_DIR ${spdlog_SOURCE_DIR}
 	SOURCE_DIR ${spdlog_SOURCE_DIR}
 	URL https://github.com/gabime/spdlog/archive/v1.3.1.tar.gz
```

Comment:
I wanted to upgrade to 1.5 but this causes issues with finding spdlog in the build folder for some reason